.github/workflows/nix.yaml

```diff
@@ -19,7 +19,7 @@ jobs:
     if: ${{ needs.skip_duplicate_jobs.outputs.should_skip != 'true' }}
     strategy:
       matrix:
-        os: [ ubuntu-latest macOS-latest ]
+        os: [ ubuntu-latest, macOS-latest ]
```

Comment:
Can we set the flake check to required before merging this in?  It would have prevented me from breaking this in the first place...

Reply:
Added.  I also added the clippy checks.